index.html

```diff
@@ -260,6 +260,8 @@ <h2>Deprecated Curriculum</h2>
            <td>None
            <td>30mins
            <td>
+            <a href="https://github.com/gdisf/teaching-materials/blob/master/_DEPRECATED/github-desktop/follow_up.md">Follow Up Email</a>
```

Comment:
While you're in here, can you fix the closing `</td>` and `</tr>` ?

Reply:
The whole index page is missing closing tags for almost everything. I wasn't sure if that was an intentional thing, but this matches that convention. Should the whole page be updated?

Reply:
Agree, the formatting that should be addressed in a separate PR. <tr>s and <td>s are self closing and I believe it was a matter of preference when @pamelafox started it. I will merge.

Reply:
Yes, I tend to leave off ending tags for </td> </tr> and <li>, to reduce tag soup. They're not required per the spec, as the browser can determine when those tags end. It's probably more popular these days to always use end tags due to frameworks like React. Go with whatever y'all prefer!